validation/process_user.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"github.com/opencontainers/runtime-tools/validation/util"
+)
+
+func main() {
+	g := util.GetDefaultGenerator()
+	g.SetProcessUID(10)
+	g.SetProcessGID(10)
+	g.AddProcessAdditionalGid(5)
```

Comment:
Should this switch on `runtime.GOOS` or an environment variable or flag for adding [`username`][1] instead?

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#windows-user

Reply:
I have fixed it  in #614.